src/test/scala/examples/Router.scala

```diff
@@ -57,8 +59,13 @@ class Router extends Module {
     io.read_routing_table_request.nodeq()
     io.load_routing_table_request.nodeq()
     io.read_routing_table_response.noenq()
+    io.read_routing_table_response.bits := 0.U
     io.in.nodeq()
-    io.outs.foreach { out => out.noenq() }
+    io.outs.foreach { out =>
```

Comment:
Can't this be done in the noenq?

Reply:
It could (in fact, at one point it did - and the comment in noenq() needs updating) but I ran into complications (I believe with generating the appropriate 0 value of the type being queued), and I reckoned we don't really need to generate additional hardware when simply setting `valid := false.b` should be sufficient. I like the symmetry with nodeq().

Also, explicitly adding a NOP connection might hide cases where the client doesn't connect the decoupled bits.